pgcli/main.py

```diff
@@ -286,12 +287,15 @@ def prompt_tokens(cli):
         with self._completer_lock:
             buf = PGBuffer(always_multiline=self.multi_line, completer=self.completer,
                     history=FileHistory(os.path.expanduser(history_file)),
-                    complete_while_typing=Always())
+                    complete_while_typing=Always(),
+                    accept_action=AcceptAction.RETURN_DOCUMENT)
 
             application = Application(style=style_factory(self.syntax_style, self.cli_style),
                                       layout=layout, buffer=buf,
                                       key_bindings_registry=key_binding_manager.registry,
                                       on_exit=AbortAction.RAISE_EXCEPTION,
+                                      on_abort=AbortAction.RETRY,
```

Comment:
This defines what happens if Control-C has been pressed. By default it's ignored.